### ISSUE TITLE
[Accessibility][Resize -Flow Chart]: Filter dialogs list is not visible when zoom is set to 200%.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2057]
**User Experience** 
Users having low vision will get difficulties to use the functionality of the page if the controls gets hide at 200%.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, set the zoom at 200% and observe the issue at filter dialog pane.

**Actual:** 
Whole filter pane dialog gets hide when user zoom the page to 200%.

**Note:**
This issue is repro throughout the application.

**Expected:**
The controls should be available despite of zoom percentage. As these are the core controls so they should be available so that the user will use the full functionality of the page.

**MAS Impacted:** [MAS1.4.4](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d})

**Attachment:**
![MAS1 4 4_Filter pane gets hide at 200%](https://user-images.githubusercontent.com/60913131/75111310-79e22600-565e-11ea-9a1d-101c7fca19b2.png)

**Comment by lei9444** [comment]
@cwhitten @ashish315 I find that this has been fixed in PR #2552. This will not happen in master now.

**Comment by ashish315** [comment]
As checked the issue is resolved, so HCL is closing the issue.
